android/airship-framework-proxy/build.gradle

```diff
@@ -84,15 +81,23 @@ def pomConfig = {
     }
 }
 
+//task sourceJar(type: Jar) {
+//    archiveClassifier.set("sources")
+//    from(android.sourceSets["main"].java.srcDirs)
+//}
+
 publishing {
     publications {
         Production(MavenPublication) {
+
+
+
             groupId project.group
             artifactId project.name
             version project.version
 
             artifact("$buildDir/outputs/aar/" + project.name + "-release.aar")
-            artifact(sourcesJar)
+//            artifact(sourcesJar)
```

Comment:
I dont think we need this but if we know the replacement i can add it. @jyaganeh

Reply:
This should be fine for now. It should only impact the team, and only when we're using IDEs smart enough to pull in the sources for docs. We can revisit if that ends up being a pain point.